api.py

```diff
@@ -412,7 +412,124 @@ def delete(self):
         self.flush()
         self.finish()
 
+class MongoHandler(tornado.web.RequestHandler):
+
+    @tornado.web.asynchronous
+    def get(self):
```

Comment:
I think this should be a post, especially if we are using a body as part of the request